### DISCUSSION TITLE
*Please* increase rate limit on the mobile app, especially for code browsing

### DISCUSSION THREAD

**Discussion by pdevito3** [community/community#6600] [Mobile]
I'm on iOS15 and almost daily when I'm browsing through GitHub I get rate limited and hit with a 'try again' screen and it kills me. 

This is usually happening when I'm quickly navigating directories in a repo, so even if it was just that it would help a ton. It happens a lot though and it totally interiors my experience and makes me lose where I'm at.

**Comment by aconsuegra** [comment]
Hi @pdevito3,
thanks for getting in touch and your feedback. We recently made some changes in the app that should help with rate limit, have you been seeing the same problem running the most up to date version of the app?

**Reply by pdevito3** [reply]
I did see it once yesterday but I think I've been seeing it a bit less recently

**Reply by aconsuegra** [reply]
Thanks for the quick response! I'm glad to know that things seem to be a bit better.

**Reply by pdevito3** [reply]
FYI, things are really bad again. See this screen record:

https://user-images.githubusercontent.com/24260530/147933466-aa0c3450-49c3-450b-a8d4-a0cf65fb0876.MOV

**Reply by badcel** [reply]
It feels like the app is refreshing the content automatically in the background even if there is no need to.

I just had the effect that I opened a yml file and the error occurred and after one second it vanished and showed the code again, and a second later the error was displayed again and so on and so on without user input.

**Reply by Sunidaponchai** [reply]
> I did see it once yesterday but I think I've been seeing it a bit less recently

**Reply by badcel** [reply]
Did you try the latest version 1.47.1? For me this version works again.

**Comment by pdevito3** [comment]
> Hi @pdevito3,
> 
> thanks for getting in touch and your feedback. We recently made some changes in the app that should help with rate limit, have you been seeing the same problem running the most up to date version of the app? 

Guys, it's coming back and it's really heavy and frustrating. I didn't even feel like I was rushing through everything. ![image](https://user-images.githubusercontent.com/24260530/143972864-66f77299-ed10-413a-8684-e70ac71082a0.png)

**Comment by Tanapl02** [comment]
I'm on iOS15 and almost daily when I'm browsing through GitHub I get rate limited and hit with a 'try again' screen and it kills me.

This is usually happening when I'm quickly navigating directories in a repo, so even if it was just that it would help a ton. It happens a lot though and it totally interiors my experience and makes me lose where I'm at.

**Reply by pdevito3** [reply]
yup same. almost always when i'm doing repo nav. maybe 80% of the time?

**Comment by badcel** [comment]
For me this is only reproduceable on repositories where I’m involved: https://github.com/github/feedback/discussions/7894

If I have the error wait a bit and browse a foreign repo it works like expected. Is it the same for you guys?

**Reply by pdevito3** [reply]
any repo for me. the rate limit expires somewhat soon, so i'm back in the running if i wait a minute or so, but at that point i'm on to something else and annoyed

**Reply by badcel** [reply]
The last update to 1.47.1 seems to have it fixed.

**Comment by TTSxvilgax** [comment]
> #- [ ] - - ![image](https://user-images.githubusercontent.com/96710718/147435256-ea0644a7-75ec-4c27-a735-0226deb82101.jpeg)[__]()**@~~~~````~~~~** ###############################################################################
# Set default behavior to automatically normalize line endings.
###############################################################################
* text=auto

###############################################################################
# Set default behavior for command prompt diff.
#
# This is need for earlier builds of msysgit that does not have it on by
# default for csharp files.
# Note: This is only used by command line
###############################################################################
#*.cs     diff=csharp

###############################################################################
# Set the merge driver for project and solution files
#
# Merging from the command prompt will add diff markers to the files if there
# are conflicts (Merging from VS is not affected by the settings below, in VS
# the diff markers are never inserted). Diff markers may cause the following 
# file extensions to fail to load in VS. An alternative would be to treat
# these files as binary and thus will always conflict and require user
# intervention with every merge. To do so, just uncomment the entries below
###############################################################################
#*.sln       merge=binary
#*.csproj    merge=binary
#*.vbproj    merge=binary
#*.vcxproj   merge=binary
#*.vcproj    merge=binary
#*.dbproj    merge=binary
#*.fsproj    merge=binary
#*.lsproj    merge=binary
#*.wixproj   merge=binary
#*.modelproj merge=binary
#*.sqlproj   merge=binary
#*.wwaproj   merge=binary

###############################################################################
# behavior for image files
#
# image files are treated as binary by default.
###############################################################################
#*.jpg   binary
#*.png   binary
#*.gif   binary

###############################################################################
# diff behavior for common document formats
# 
# Convert binary document formats to text before diffing them. This feature
# is only available from the command line. Turn it on by uncommenting the 
# entries below.
###############################################################################
#*.doc   diff=astextplain
#*.DOC   diff=astextplain
#*.docx  diff=astextplain
#*.DOCX  diff=astextplain
#*.dot   diff=astextplain
#*.DOT   diff=astextplain
#*.pdf   diff=astextplain
#*.PDF   diff=astextplain
#*.rtf   diff=astextplain
#*.RTF   diff=astextplain

**Comment by HANOSSAH** [comment]
نسيت كلمت السر
